pysatMissions/methods/spacecraft.py

```diff
@@ -0,0 +1,235 @@
+"""Provides default routines for projecting values onto vectors
+for pysat instruments.
+
+"""
+
+import numpy as np
+import pysatMagVect
+
+
+def add_sc_attitude_vectors(inst):
```

Comment:
Since this is specific to ram pointing, should we add that to the method name?
add_ram_pointing_sc_attitude_vectors

Reply:
fixed